src/media/css/buttons.styl

```diff
@@ -138,33 +126,38 @@ a.button {
     }
 }
 
-// Install buttons.
-.button.install {
-    // Unset many .button properties to compensate for larger click target.
+.mkt-app-button {
+    align-items: center;
     btn_color(transparent, transparent, transparent, false);
+    border-radius: 5px;
     cursor: pointer;
+    display: flex;
     font-size: $btn-install-font-size;
-    height: 34px;  // Actual click-target height. Queen Krupa request.
-    min-width: $btn-install-width;
-    padding: 0;
-    position: absolute;
-    right: 0;
+    height: 34px;
+    margin-top: 8px;
+    width: fit-content;
 
+    &.button {
+        padding: 0;
+    }
     +rtl() {
         left: 0;
         right: inherit;
     }
     em {
         // This em becomes the visual button.
-        btn_color($action-positive, $action-positive-tapped, $action-positive-hover, true);
+        btn_color($btn-color, $action-positive-tapped,
+                  $action-positive-hover, false);
```

Comment:
I think it's fine to wrap arguments within a Stylus mixin call? For plain rules, you need to backslash at EOL.